### ISSUE TITLE
amuse-framework 2021.3.0 installation fails

### ISSUE THREAD

**Issue by paulkerry1** [amusecode/amuse#741]
Hi

I tried to install using "pip3 install amuse-framework" which failed, then manually and got the following error (same error for either method)...

```
python3 setup.py build
Traceback (most recent call last):
  File "setup.py", line 71, in <module>
    scripts=[ "bin/amusifier" ],
  File "/usr/local/lib/python3.7/dist-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.7/distutils/core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/local/lib/python3.7/dist-packages/setuptools/dist.py", line 433, in __init__
    k: v for k, v in attrs.items()
  File "/usr/lib/python3.7/distutils/dist.py", line 292, in __init__
    self.finalize_options()
  File "/usr/local/lib/python3.7/dist-packages/setuptools/dist.py", line 709, in finalize_options
    ep(self)
  File "/usr/local/lib/python3.7/dist-packages/setuptools/dist.py", line 716, in _finalize_setup_keywords
    ep.load()(self, ep.name, value)
  File "/usr/local/lib/python3.7/dist-packages/setuptools_scm/integration.py", line 26, in version_keyword
    dist.metadata.version = _get_version(config)
  File "/usr/local/lib/python3.7/dist-packages/setuptools_scm/__init__.py", line 185, in _get_version
    template=config.write_to_template,
  File "/usr/local/lib/python3.7/dist-packages/setuptools_scm/__init__.py", line 96, in dump_version
    with open(target, "w") as fp:
FileNotFoundError: [Errno 2] No such file or directory: '../../src/amuse/version.py'
```
Changing setup.py line 47 in the source directory from...
`"root":` `"../..",`
to
`"root":` `".",`

makes it build and install OK.

The same problem then continues to appear when installing individual amuse packages...
amuse-athena-2021.3.0
    FileNotFoundError: [Errno 2] No such file or directory: '../../src/amuse/community/athena/version.py'
amuse-bhtree-2021.3.0
    FileNotFoundError: [Errno 2] No such file or directory: '../../src/amuse/community/bhtree/version.py'
etc, etc,

**Environment (please complete the following information):**
 - Debian 10 amd64
 - Python 3.7.3
 - pip 21.0.1
 - setuptools-scm 5.0.1

**Comment by rieder** [comment]
Thanks for reporting. This is slightly more complicated than just changing that line I fear, but I hope to have a fix today.

**Comment by rieder** [comment]
Now fixed in release 2021.3.1

**Comment by paulkerry1** [comment]
All works perfectly - thanks very much for the quick fix.

Cheers
Paul.
